frontend/app/components/ToolCard.tsx

```diff
@@ -63,17 +63,20 @@ export default function ToolCard (props: ToolCardProps) {
       <CardActions>
         <Grid container justifyContent='space-between'>
           {tool.navigation_enabled ? <RemoveToolButton /> : <AddToolButton />}
-          <IconButton
-            onClick={() => setLearnMoreActive(!learnMoreActive)}
-            aria-expanded={learnMoreActive}
-            aria-label={`${learnMoreActive ? 'Close ' : ''}Learn More`}
-            size='large'
+          <Button
+            onClick={() => setShowMoreInfo(!showMoreInfo)}
+            aria-expanded={showMoreInfo}
+            aria-label={`Show ${!showMoreInfo ? 'More' : 'Less'} Info`}
```

Comment:
The label content is different from those proposed in #146 Maybe an update in #146 description is needed?

Reply:
Should the font size of label "MORE"/"LESS" links be the same as the "ADD TOOL"/"REMOVE TOOL" button text?

Reply:
They should be the same, good observation. I think because I'm using a `Typography` component inside `Button` for the tool buttons, it's making it slightly bigger. I'll remove those `Typography` components for consistency.

Reply:
Okay, take a look at https://github.com/tl-its-umich-edu/canvas-app-explorer/pull/200/commits/30bb7662b8c27401467b86f7308f34e272ece19d. Font size should be resolved.

Reply:
I've updated the main comment and title in #146 with more info explaining what I did here and how what feedback was given from other team members.